nautilus_trader/data/aggregation.pyx

```diff
@@ -907,14 +951,22 @@ cdef class TimeBarAggregator(BarAggregator):
                 f"Aggregation not time based, was {bar_aggregation_to_str(aggregation)}",
             )
 
+    cpdef void _invalidate_skip_first_non_full_bar_on_exact_start(self, datetime now, datetime start_time):
```

Comment:
Are we sure this isn't something solvable elsewhere, or relevant only for backtests?